src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/BodySubstitution.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Internal.IL;
+using Internal.IL.Stubs;
+using Internal.TypeSystem;
+
+namespace ILCompiler
+{
+    internal class BodySubstitution
```

Comment:
```suggestion
    internal sealed class BodySubstitution
```
I think we can seal this